generators/client/templates/angular/src/main/webapp/swagger-ui/index.html.ejs

```diff
@@ -102,6 +102,10 @@
                     if (authToken) {
                         req.headers['Authorization'] = "Bearer " + authToken;
                     }
+                    // Remove the sample Swagger UI request body if present
+                    if (req.body === '{"additionalProp1":"string","additionalProp2":"string","additionalProp3":"string"}') {
```

Comment:
you need to add this part above too, for oauth2 configuration

Reply:
done